docs/tutorials/getting_started/tutorial_overview.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Getting started with Great Expectations
+---
+
+Welcome to the Great Expectations getting started tutorial! This tutorial will help you set up your first local deployment of Great Expectations.  This deployment will contain a small Expectation Suite that we will use to validate some sample data. We'll also introduce important concepts, with links to detailed material you can dig into later.
+
+:::tip
+The steps described in this tutorial assume you are installing Great Expectations version 0.13.8 or above.
+
+For a tutorial for older versions of Great Expectations, please see older versions of this documentation, which can be found [here](https://docs.greatexpectations.io/en/latest/guides/tutorials.html).
+:::
+
+### This tutorial will walk you through the following steps
+
+<table class="borderless">
```

Comment:
Is there a reason this is done using HTML instead of a markdown table?

Reply:
To the best of my knowledge the markdown tables don't permit assigning a class for CSS styling, so by Docusaurus' default using a markdown table would include borders and alternating background colors for rows.